components/desktop/mate/mate-session-manager/Makefile

```diff
@@ -34,6 +34,10 @@ include $(WS_MAKE_RULES)/common.mk
 
 # Build requirements
 REQUIRED_PACKAGES += x11/library/xtrans
+REQUIRED_PACKAGES += developer/build/autoconf-archive
+
+## A note on nvidia dependencies. In order to things to properly depend on libGL or libEGL we have a line in the manifest.
+## Whoever rebuilds this will need to remove the REQUIRED_PACKAGES lines generated manually as we have no filtering mechanism
```

Comment:
This sounds really strange.  Could you please elaborate what is this all about?  For example, what you mean by _filtering mechanism_?

Reply:
We need to depend on any nvidia so people can install any graphics driver. ogl-select selects it at runtime and we have no way to properly handle require-any dependencies in the package that cannot be installed together.